rest/authorization-server/src/main/java/org/eclipse/sw360/rest/authserver/Sw360AuthorizationServer.java

```diff
@@ -11,32 +11,51 @@
 
 package org.eclipse.sw360.rest.authserver;
 
+import org.apache.log4j.Logger;
 import org.eclipse.sw360.datahandler.common.CommonUtils;
 import org.eclipse.sw360.datahandler.thrift.users.UserGroup;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.web.support.SpringBootServletInitializer;
 
+import javax.crypto.SealedObject;
+import java.io.IOException;
 import java.util.Properties;
 
+import static org.eclipse.sw360.rest.authserver.security.Sw360SecurityEncryptor.encrypt;
+
 @SpringBootApplication
 public class Sw360AuthorizationServer extends SpringBootServletInitializer {
 
     private static final String PROPERTIES_FILE_PATH = "/sw360.properties";
     private static final String DEFAULT_ACCESS_TIME_IN_SECONDS = "3600";
     private static final String DEFAULT_WRITE_ACCESS_USERGROUP = UserGroup.SW360_ADMIN.name();
 
-    public static final int TOKEN_ACCESS_VALIDITY;
-    public static final UserGroup WRITE_ACCESS_USERGROUP;
+    public static final int CONFIG_TOKEN_ACCESS_VALIDITY;
+    public static final UserGroup CONFIG_WRITE_ACCESS_USERGROUP;
+    public static final String CONFIG_CLIENT_ID;
+    public static final SealedObject CONFIG_CLIENT_SECRET;
+
+    private static final Logger log = Logger.getLogger(Sw360AuthorizationServer.class);
 
     static {
         Properties props = CommonUtils.loadProperties(Sw360AuthorizationServer.class, PROPERTIES_FILE_PATH);
-
-        TOKEN_ACCESS_VALIDITY = Integer.parseInt(props.getProperty(
+        CONFIG_TOKEN_ACCESS_VALIDITY = Integer.parseInt(props.getProperty(
                 "rest.token.access.validity", DEFAULT_ACCESS_TIME_IN_SECONDS));
```

Comment:
Variable names do not match. If the variable is called `CONFIG_TOKEN_ACCESS_VALIDITY` I would expect the default value to be called `DEFAULT_TOKEN_ACCESS_VALIDITY`.

But then you loos the `_IN _SECONDS` postfix. and the name is less descriptive. Maybe needs a comment.

Reply:
Changed to `DEFAULT_TOKEN_ACCESS_VALIDITY` and added a comment (validity time in seconds)

Reply:
`CONFIG_CLIENT_SECRET` is defined as final constant.
I can only initialize this constant value for one time. (in constructor)
To consider a correct exception handling I need to create this temp variable `clientSecret`.
I renamed `clientSecret` to `tmpClientSecret`.